### ISSUE TITLE
Label translation

### ISSUE THREAD

**Issue by Phocea** [MhdSyrwan/aor-embedded-array#2]
Hello,

Any news about the label translation which is currently not working?
I think you were waiting on some code change in aor?

Thanks

**Comment by MhdSyrwan** [comment]
Yes exactly.
They have not replied yet !

What you can do currently is to put labels manually.

**Comment by Phocea** [comment]
ok..might be worth raising a new PR for this as the initial one may be a bit out of sight!

**Comment by MhdSyrwan** [comment]
I will open a PR soon. but the problem is that some parts of the patch have no general meaning at all. They will be just written to make this library work ! They will not accept something like this.

That's why I wanted to see if there's a more general thing to be added to core before opening the PR.

The part I'm talking about is the part that replaces any `[number].` with `.fields.` like `links[3].context` => `links.fields.context`
